### ISSUE TITLE
property "all-labels" is not defined in object type {} on actions/labeler@v4

### ISSUE THREAD

**Issue by EigoOda** [reviewdog/action-actionlint#105]
```yaml
...
  labeler:
    runs-on: ubuntu-latest
    steps:
      - id: labeler
        uses: actions/labeler@v4
        with:
          repo-token: ${{ secrets.GITHUB_TOKEN }}
          configuration-path: .github/labeler.yaml
          sync-labels: ''
    outputs:
      labels: ${{ steps.labeler.outputs.all-labels }}
```
actionlint says 'property "all-labels" is not defined in object type {} [expression]'.

But the Following jobs (need labeler job outputs) are running correctly.
How can I resolve this warning?
Thanks.

**Comment by EigoOda** [comment]
post to https://github.com/rhysd/actionlint/issues/346

**Comment by rahimme2k7** [comment]
DO YOU GET ANY SOLUTION REGARDING THIS ISSUE. I AM FACING SILIAR PROBLEM.
